react-sdk/src/providers/tambo-thread-provider.tsx

```diff
@@ -743,7 +741,7 @@ export const TamboThreadProvider: React.FC<
         threadId = currentThreadId ?? PLACEHOLDER_THREAD.id,
         streamResponse = streaming,
         forceToolChoice,
-        contextKey = currentThreadContextKey,
+        contextKey,
```

Comment:
this basically means the user has to pass in contextKey themselves, but that was kind of the design of the API to begin with anyway